wrappers/node/test/suite1/ariesvcx-connection.test.ts

```diff
@@ -24,25 +34,20 @@ describe('Connection:', () => {
   describe('connect:', () => {
     it('success: without phone', async () => {
       const connection = await connectionCreate()
-      const inviteDetails = await connection.connect({ data: '{"connection_type":"QR"}' })
+      const inviteDetails = await connection.connect({ data: '{}' })
       assert.notEqual(inviteDetails, '')
     })
-
-    it('success: with phone', async () => {
-      const connection = await connectionCreate()
-      const inviteDetails = await connection.connect({ data: '{"connection_type":"SMS","phone":"7202200000"}' })
-      assert.notEqual(inviteDetails, '')
-    })
-
+  //
     it('throws: not initialized', async () => {
       const connection = new (Connection as any)()
       const err = await shouldThrow(async () => connection.connect({ data: '{"connection_type":"QR"}' }))
       assert.equal(err.vcxCode, VCXCode.INVALID_CONNECTION_HANDLE)
     })
```

Comment:
What is QR connection type? Isn't the test description incorrect?

Reply:
Some legacy stuff, the `data` option is not used when running with aries. So I'll remove the QR option here. The test name is unmodified and still passing. I think the test name is correct  - test is supposed to failed because I think using `new (Connection as any)()` instantiation it never calls into FFI, only create a dumb typescript object.